src/app/App.module.css

```
@@ -0,0 +1,55 @@
+.container {
+  font-family: 'Schoolbell', cursive;
+  display: grid;
+  grid-template-rows: 45% 1fr 1fr;
+  text-align: center;
+  background-image: url(https://images.unsplash.com/photo-1517263904808-5dc91e3e7044?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=388&q=80);
+  background-repeat: y-repeat;
+  background-size: 100vw;
+  height: 100vh;
+}
+
+.form {
+  background-color: rgb(3, 7, 13, 75%);
+  margin: 7.5rem;
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr;
+  gap: 0.5rem;
+  justify-items: center;
+  align-content: space-around;
+  border: dotted #9bf2d2 3px;
+  border-radius: 25%;
+}
+
+.options {
+  color: #9bf2d2;
+  margin-top: 1rem;
+}
+
+.selection {
+  font-family: 'Schoolbell', cursive;
+  border: solid #9bf2d2 2px;
+  border-radius: 0.25rem;
+}
+
+.inputField {
+  font-family: 'Schoolbell', cursive;
+  color: dimgray;
+  border: solid #9bf2d2 2px;
+  border-radius: 0.25rem;
+}
+
+.button {
+  background-color: #9bf2d2;
```

```suggestion
  background-color: #9bf2d2;
```
This color seems to appear quite a few times, you should definitely make it a global variable 🔢